### PR TITLE
docs(completion): incompatibility

### PR DESCRIPTION
**Pull request by KSXGitHub**

### CODE REVIEW

File: docs/completion.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ id: completion
 title: Command line tab-completion
 ---
 
+:::info
+
+Completion for pnpm v9 is incompatible with completion for older pnpm versions.
+If you already install pnpm completion for version older than v9, you must uninstall it first so that completion for v9 may work properly.
```
Comment (Privat33r-dev) [inline docs/completion.md:9]: If you wanted to split on paragraphs, you need to add a newline between them

File: docs/completion.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ id: completion
...
+You may do so by removing section of code that contains `__tabtab` in your dot files.
```
Comment (Privat33r-dev) [inline docs/completion.md:10]:
```suggestion
Completion for pnpm v9 is incompatible with completion for older pnpm versions.

If you have already installed pnpm completion for a version older than v9, you must uninstall it first to ensure that completion for v9 works properly.

You can do this by removing the section of code that contains `__tabtab` in your dot files.
```

I did not check the content of the message, it's just a small suggestion to improve grammar and readability.

File: docs/completion.md (hunk 1 above)
Comment (KSXGitHub) [reply docs/completion.md:9]: I don't actually want to create multiple paragraphs. I just want to source code to look nicer in the code editor.

File: docs/completion.md [hunk 3]
```diff
@@ -3,6 +3,16 @@ id: completion
 title: Command line tab-completion
 ---
 
+:::info
+
+Completion for pnpm v9 is incompatible with completion for older pnpm versions.
+
+If you have already installed pnpm completion for a version older than v9, you must uninstall it first to ensure that completion for v9 works properly.
+
+You can do this by removing the section of code that contains `__tabtab` in your dot files.
```
Comment (zkochan) [inline docs/completion.md:12]:
```suggestion
Completion for pnpm v9 is incompatible with completion for older pnpm versions.
If you have already installed pnpm completion for a version older than v9, you must uninstall it first to ensure that completion for v9 works properly.
You can do this by removing the section of code that contains `__tabtab` in your dot files.
```

**Review by zkochan — APPROVED** [review]

**Pull request merged**